.connecment/proposal.md

```diff
@@ -0,0 +1,94 @@
+# ConnecMent Proposal: Arbit
+> Note: This proposal is for [ConnecMent](https://github.com/ConnecMent) only.
+
+## TL;DR;
+- A team of 3-4 members
+- Average 2 hours of mentees time commitment for 15 days
+- An arbitrage app (NextJS SSR), simply displaying some data to the user
+
+## Team info
+- **Owner**: @fatemeh-ra
+- **Mentors**: @mkermani144, @fatemeh-ra
+- **Mentees**: @SeyedMojtaba1
+
+## What is Arbit?
+### A metaphor
+Suppose two cities: Abunland, in which there are lots of apples, and Starvland, in which apples are very rare. You, as a magician, has the ability to teleport between the two. Other than a magician, you know a bit about economy and markets. You want to make some money. What do you do?
+Apple, in Abunland, costs you $1, but $10 in Starvland. You have $100, so you simply buy 100 apples in Abunland, teleport to Starvland, and sell them for $1000. Congrats! You made $900.
+After a while, thanks to your teleportations, an equilibrium occurs. Starvland becomes full of apples, and its cost is reduced to $3. On the other hand, in Abunland, the number of apples reduces, so its cost increases to, say, $6.
+As long as there is a mismatch in apple price (or more precisely, apple/usd pair) in Abunland or Starvland markets, you can make a profit by buying cheap in one market, and selling expensive in the other. This is called "arbitrage".
+
+### Arbit description
+Arbit is an app that computes and illustrates arbitrage opportunities based on a token pair price (from a limited list of tokens for MVP) in different markets. For MVP, only [Ergo dex](ergodex.io) and [Splash](splash.trade) are used. It computes the most profitable change amount based on pools' liquidity on backend, and shows arbitrage profits (in USD) for different token swaps on frontend.
```

Comment:
Arbitrage is somehow a general term. It's important to clarify we are not going to transfer tokens from one market to another. We are going to have the same token pairs in different markets simultaneously.

Reply:
#19